tardis/transport/montecarlo/formal_integral.py

```diff
@@ -14,14 +14,13 @@
 from tardis.transport.montecarlo import njit_dict, njit_dict_no_parallel
 from tardis.transport.montecarlo.numba_interface import (
     opacity_state_initialize,
-    NumbaModel,
     OpacityState,
 )
 from tardis.transport.montecarlo.formal_integral_cuda import (
     CudaFormalIntegrator,
 )
 
-from tardis.transport.montecarlo.spectrum import TARDISSpectrum
+from tardis.spectrum import TARDISSpectrum
 
 C_INV = 3.33564e-11
```

Comment:
Can we use astropy constants here?

Reply:
Units?

Reply:
I would like to, but it then would likely break tests. It could probably use a comment though.

Reply:
Please make an issue.

Reply:
#2616